routes/web.php

```diff
@@ -182,36 +182,60 @@
             Route::as('frontend.')->middleware(['track-visited-url'])->group(function () {
                 Route::resource('help', Cooperation\Frontend\HelpController::class)->only('index');
                 Route::as('tool.')->group(function () {
+
+
                     $scans = \App\Helpers\Cache\Scan::allShorts();
                     // TODO: Deprecate to whereIn in L9
-                    Route::get('{scan}', [ScanController::class, 'show'])
-                        ->name('scans.show')
+                    Route::get('{scan}', [Cooperation\Frontend\Tool\ScanController::class, 'redirect'])
+                        ->name('scan.redirect')
                         ->where(collect(['scan'])
-                            ->mapWithKeys(fn ($parameter) => [$parameter => implode('|', $scans)])
+                            ->mapWithKeys(fn($parameter) => [$parameter => implode('|', $scans)])
                             ->all()
                         );
 
+                    Route::prefix('{scan}/{step:slug}')
+                        ->where(collect(['scan'])
+                            ->mapWithKeys(fn($parameter) => [$parameter => implode('|', $scans)])
+                            ->all()
+                        )
+                        ->as('simple-scan.')
+                        ->group(function () {
+
+                            // Define this route as last to not match above routes as step/sub step combo
+                            Route::get('{subStep:slug}', [Cooperation\Frontend\Tool\SimpleScanController::class, 'index'])
+                                ->name('index')
+                                ->middleware(['checks-conditions-for-sub-steps', 'duplicate-data-for-user']);
+
+                            Route::get('vragenlijst/{questionnaire}', [Cooperation\Frontend\Tool\QuickScan\QuestionnaireController::class, 'index'])
+                                ->name('questionnaires.index');
+                        });
+
+                    Route::as('my-plan.')->prefix('woonplan')->group(function () {
+                        Route::get('', [Cooperation\Frontend\Tool\QuickScan\MyPlanController::class, 'index'])->name('index');
+                        Route::get('bestanden/{building?}', [Cooperation\Frontend\Tool\QuickScan\MyPlanController::class, 'media'])->name('media');
+                    });
+
+
+//                    Route::permanentRedirect('quick-scan/woonplan', 'woonplan');
```

Comment:
I see some TODO stuff here; not sure if you're aware since no TODOs in the code. It feels as if the action plan will be removed from the quick scan (which makes sense) but isn't fully implemented yet.